lib/resources/key_rsa.rb

```diff
@@ -26,7 +26,14 @@ class RsaKey < Inspec.resource(1)
     def initialize(keypath, passphrase = nil)
       @key_path = keypath
       @passphrase = passphrase
-      @key = OpenSSL::PKey.read(read_file_content(@key_path, allow_empty: true), @passphrase)
+      if @passphrase.is_a? Inspec::Attribute::DEFAULT_ATTRIBUTE
+        return fail_resource 'Please provide default value for attribute'
+      end
+      begin
```

Comment:
We should also consider moving the logic out of the initialize method, so that we make sure it is a control error and its not happening during initialization.